app/components/candidate_interface/subject_knowledge_review_component.rb

```diff
@@ -31,8 +32,13 @@ def subject_knowledge_form_row
         key: t('application_form.personal_statement.subject_knowledge.key'),
         value: @subject_knowledge_form.subject_knowledge,
         action: t('application_form.personal_statement.subject_knowledge.change_action'),
-        change_path: candidate_interface_edit_subject_knowledge_path,
+        change_path: candidate_interface_edit_subject_knowledge_path(return_to_params),
+        data_qa: 'subject-knowledge',
       }
     end
+
+    def return_to_params
```

Comment:
A lot of the components have or will have this `return_to_params` method - I can extract in another PR